src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,571 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Preload video and audio for faster playback.
+
+{# wf_published_on: 2017-07-05 #}
+{# wf_updated_on: 2017-07-05 #}
+
+# Fast Playback with Video Preload {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+<video controls controlsList="nodownload" muted playsinline style="width: 100%">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.webm#t=0.7"
+          type="video/webm">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.mp4#t=1"
+          type="video/mp4">
+</video>
+
+Faster playback means more people watching your video. That's a known fact.
+I'll explore in this article different techniques you can use to accelerate
+your media playback by actively preloading resources depending on your use
+case.
+
+Note: Unless specified otherwise, this article also applies to the audio element.
+
+### TL;DR {: .hide-from-toc }
+
+<table>
+  <tbody>
+    <tr>
+      <th></th>
+      <th>
+It's great...
+      </th>
+      <th>
+But...
+      </th>
+    </tr>
+    <tr>
+      <td rowspan=3 style="white-space: nowrap">
+<a href="#video_preload_attribute">Video preload attribute</a>
+      </td>
+      <td rowspan=3>
+Simple to use for an unique file hosted on a web server
+      </td>
+      <td>
+Browser may completely ignore the attribute
+      </td>
+    <tr>
+      <td>
+Resource fetching starts when HTML document has been completely loaded and
+parsed
+      </td>
+    </tr>
+    <tr>
+      <td>
+MSE is not compatible
+      </td>
+    </tr>
+    <tr>
+      <td rowspan=2 style="white-space: nowrap">
+<a href="#link_preload">Link preload</a>
+      </td>
+      <td>
+Forces the browser to make a request for a video resource without blocking
+the document's <code>onload</code> event
+      </td>
+      <td>
+HTTP Range requests are not compatible
+      </td>
+    <tr>
+      <td>
+Compatible with MSE and segment files
+      </td>
+      <td>
+Should be used only for audio files or short video clips when fetching full
+resource
+      </td>
+    </tr>
+    <tr>
+      <td>
+<a href="#manual_buffering">Manual buffering</a>
+      </td>
+      <td>
+Full control
+      </td>
+      <td>
+Complex error handling is website responsibility
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Video preload attribute
+
+If the video source is an unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload]. This means [Media Source Extensions
+(MSE)] is not compatible with `preload`.
+
+Resource fetching will start only when initial HTML document has been
+completely loaded and parsed (eg. `DOMContentLoaded` event) while the very
+different `load` event will be fired when resource has actually been fetched.
```

Comment:
IIUC a key point here is that the preload attribute only works within the context of the current page, unlike link preload which can work on other pages (warming up the video before the user navigates to the page with the player). Please call that out specifically, it's an important reason why people might choose between the two approaches.